demos/storybook/stories/app-bar/with-basic-usage.tsx

```diff
@@ -3,17 +3,21 @@ import { number } from '@storybook/addon-knobs';
 import { StoryFnReactReturnType } from '@storybook/react/dist/client/preview/types';
 import { WITH_MIN_PROPS_STORY_NAME } from '../../src/constants';
 import { AppBar } from '@pxblue/react-components';
-import { bodyFiller } from './filler';
+import { Toolbar, Typography } from '@material-ui/core';
+import { getBodyFiller } from '../../src/utils';
 
 export const withBasicUsage = (): StoryFnReactReturnType => {
     const expandedHeight = number('expandedHeight', 200);
     const collapsedHeight = number('collapsedHeight', 64);
+    const scrollThreshold = number('scrollThreshold', 136);
```

Comment:
The height knobs aren't taking effect right away (until the user scrolls). We might need to add another useEffect to listen for height changes (I had to do this on the RN side).

Reply:
Seems like it is only an issue for the snap variant.

Reply:
Took a stab at fixing this too. Works fine for the collapsedHeight, but not so swift for the expandedHeight due to the way we are calculating the isExpanded variable.